src/solve_function.jl

```diff
@@ -13,3 +12,23 @@ function SciMLBase.solve(cache::LinearCache, alg::LinearSolveFunction,
 
     return SciMLBase.build_linear_solution(alg, cache.u, nothing, cache)
 end
+
+struct ApplyLdiv <: AbstractSolveFunction end
+function SciMLBase.solve(cache::LinearCache, alg::ApplyLdiv, args...; kwargs...)
+    @unpack A, b, u = cache
+
+    v = A \ b
+    copy!(u, v)
+
+    return SciMLBase.build_linear_solution(alg, cache.u, nothing, cache)
+end
```

Comment:
oh, the naming is weird. `DirectLdiv(inplace = true)`?

Reply:
that works haha